torch/testing/_creation.py

```diff
@@ -0,0 +1,159 @@
+"""
+This package contains tensor creation utilities.
+"""
+
+import torch
+from typing import Optional, Tuple, Union
+import math
+
+__all__ = [
+    "make_tensor"
+]
+
+def make_tensor(
+    size: Tuple[int, ...],
+    *,
+    device: Union[str, torch.device] = "cpu",
+    dtype: torch.dtype = torch.float32,
+    low: Optional[float] = None,
+    high: Optional[float] = None,
+    requires_grad: bool = False,
+    noncontiguous: bool = False,
+    exclude_zero: bool = False
+) -> torch.Tensor:
+    r"""Creates a random tensor with the given :attr:`size`.
```

Comment:
Let's rename size->shape to be consistent with NumPy (and our own shape operator)

In NumPy size is actually the # elements of the array